### PR TITLE
 Slack Support URL in action buttons

### PR DESCRIPTION
**Pull request by kur4tor**
more information: https://api.slack.com/docs/message-attachments#link_buttons (Button attachment format)

### CODE REVIEW

File: src/AttachmentAction.php [hunk 1]
```diff
@@ -151,6 +159,23 @@ public function setValue(?string $value): self
 
         return $this;
     }
+    
+    /**
+     * @param string $url
+     *
+     * @return AttachmentAction
+     */    
+    public function setUrl(string $url): self
```
Comment (soullivaneuh) [inline src/AttachmentAction.php]: If the url can be null, you must put `?string`.

File: src/AttachmentAction.php [hunk 2]
```diff
@@ -151,6 +159,23 @@ public function setValue(?string $value): self
...
+    {
+        $this->url = $url;
+
+        return $this;
+    }
+
+    public function getUrl(): string
```
Comment (soullivaneuh) [inline src/AttachmentAction.php]: `?string`

**Review by soullivaneuh — CHANGES_REQUESTED** [review]
Thanks for the PR!

I left some review comments.

**Comment by soullivaneuh** [comment]
Also, Travis was failing. Could you please take a look? Thanks!

**Comment by kur4tor** [comment]
I do not know if slack API support it when null variables are sent to him. I close and will send new Pull request included tests and if i will really sure what is supported

**Comment by soullivaneuh** [comment]
Hmm... why closing the PR? :thinking:?

**Comment by soullivaneuh** [comment]
Just saw your edited comment. You may just keep this one open and push your modification if you find something new. :+1: 

Regards.

**Comment by kur4tor** [comment]
I must test in fork master branch first.

**Pull request closed without merging**